packages/malloy-render/package.json

```diff
@@ -24,6 +24,7 @@
     "clean": "tsc --build --clean",
     "prepublishOnly": "npm run build",
     "storybook": "rm -rf ./node_modules/.cache && storybook dev -p 6006",
+    "storybook-windows": "del /s /q .\\node_modules\\.cache && storybook dev -p 6006",
```

Comment:
just a bystander comment: [rimraf](https://www.npmjs.com/package/rimraf) is what often gets used for cross-platform removal of old stuff. You'd use `rimraf` as CLI commands like [in this random example](https://github.com/observablehq/framework/blob/main/package.json#L25)

Reply:
@srenatus thanks for the tip! wish you had beat me to the review.

Reply:
It was merely a chance find on my end. I hadn't been looking at Malloy in a long while 😅